modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json

```diff
@@ -4547,8 +4547,13 @@
                     "id": "1",
                     "type": "intent_to_file",
                     "attributes": {
+<<<<<<< HEAD
+                      "creationDate": "2023-08-23",
+                      "expirationDate": "2024-08-23",
+=======
                       "creationDate": "2023-08-22",
                       "expirationDate": "2024-08-22",
+>>>>>>> origin/master
```

Comment:
looks like a conflict resolution got missed

Reply:
Good catch, thanks! Resolved manually but neglected to save.